src/detours.cpp

```diff
@@ -52,31 +52,89 @@ namespace rhost {
         }
 
         // MessageBoxW
-        decltype(MessageBoxW) *pMessageBoxW = NULL;
+        decltype(MessageBoxW) *pMessageBoxW = nullptr;
         int WINAPI DetourMessageBoxW(HWND hWnd, LPCWSTR lpText, LPCWSTR lpCaption, UINT uType) {
             char ch[1000];
             wcstombs(ch, lpText, _countof(ch));
             return HostMessageBox(ch, uType);
         }
 
         // MessageBoxA
-        decltype(MessageBoxW) *pMessageBoxA = NULL;
+        decltype(MessageBoxA) *pMessageBoxA = nullptr;
```

Comment:
MBA and MBW eventually call a internal W function. MBA does not internally call MBW.